### ISSUE TITLE
Mutiny example is incorrect

### ISSUE THREAD

**Issue by jponge** [r2dbc/r2dbc.github.io#25]
Hi,

Looking at the Mutiny example on the front page there are many issues, including:

```java
ConnectionFactory connectionFactory = ConnectionFactories
  .get("r2dbc:h2:mem:///testdb");
Uni.createFrom().publisher(connectionFactory.create())
.toMulti()
.flatMap(connection -> connection
.createStatement("SELECT firstname FROM PERSON WHERE age > $1")
.bind("$1", 42)
.execute())
.flatMap(result -> result
.map((row, rowMetadata) -> row.get("firstname", String.class)))
.on()
.item().invoke(firstname -> System.out.println(firstname))
.subscribe();
```

1. Instead of starting from a `Uni` converted to a `Multi`, you could start with `Multi.createFrom().(...)`
2. we do have `flatMap`, but `onItem().transformTo{...}` makes for more readable code,
3. `onItem().invoke(...)` can indeed be used to consume elements, but you should do it  in the subscription final handler,
4. speaking of the subscription this code for sure does nothing, `subscribe()` must be followed by a subscription, like `.subscribe().with(...)` / `.subscribe().withSubscriber(....)`, etc.

Is there any actual example somewhere to compile  and test what you intend to show on the front page?

As it is now the Mutiny sample lacks correctness and idiomatic usage of the APIs  we provide.

**Comment by mp911de** [comment]
Thanks for raising an issue. The code on the site does not have a compiled variant as it is a pure Jekyll site. How about submitting a pull request to align the sample with how Mutiny is intended to be used?

**Comment by jponge** [comment]
Do you have Mutiny bindings or examples somewhere in the code base so I could check out, test and propose something idiomatic?

**Comment by mp911de** [comment]
While R2DBC isn't tied to a particular implementation of Reactive Streams, we have a bit of TCK code at [`TestKit`](https://github.com/r2dbc/r2dbc-spi/blob/main/r2dbc-spi-test/src/main/java/io/r2dbc/spi/test/TestKit.java#L567-L581) that requires a concrete Reactive Streams implementation. The [MySQL](https://github.com/mirromutth/r2dbc-mysql/blob/main/src/test/java/dev/miku/r2dbc/mysql/MySqlTextTestKit.java) and [Postgres](https://github.com/pgjdbc/r2dbc-postgresql/blob/main/src/test/java/io/r2dbc/postgresql/PostgresqlTestKit.java) drivers allow a quick-start by requiring Docker and Testcontainers. With either one you should be able to give it a spin with Mutiny.

The samples on the website are examples to give users an idea of how the SPI is used in combination with a composition library. The motivation for `Uni.createFrom().publisher(…).multi(…)` was to illustrate that `ConnectionFactory` emits a single item that then creates a stream of multiple items (Results).

**Comment by jponge** [comment]
Noted thanks, I will have a look and get back to you.

**Comment by jponge** [comment]
@mp911de I see that `index.md` uses embedded HTML, do you have a quick CLI command to generate the pile of HTML `div`/`span`?

The Mutiny samples should be:

```java
Uni.createFrom().publisher(connectionFactory.create())
  .onItem().transformToMulti(connection -> connection
    .createStatement("SELECT firstname FROM PERSON WHERE age > $1").bind("$1", 42).execute())
  .onItem().transform(result -> result.map((row, rowMetadata) -> row.get("firstname", String.class)))
  .subscribe().with(System.out::println);
```

**Comment by mp911de** [comment]
Thanks. Formatting the code with http://rouge.jneen.net/v3.25.0/java/ should do the trick.

**Comment by jponge** [comment]
A quick Ruby script did the job, see #26.

**Comment by mp911de** [comment]
Fixed via #26.

**Comment by mp911de** [comment]
Thank you for your contribution. That's merged now. I updated the other samples to use `System.out.println(…)` via method reference to align the samples.
